packages/gestalt/src/Badge.css

```diff
@@ -1,19 +1,27 @@
+:root {
+  --font-size-md: 12px;
+  --height: 24px;
+  --width: 42px;
+}
+
 .Badge {
   composes: antialiased sansSerif from "./Typography.css";
-  composes: fontWeightBold from "./Typography.css";
-  composes: blue from "./Colors.css";
   composes: inlineBlock from "./Layout.css";
-  border-radius: 4px;
-  border-style: solid;
-  border-width: 1px;
-  font-size: 10px;
-  line-height: 1;
+  composes: rounding2 from "./Borders.css";
+  composes: white from "./Colors.css";
+  composes: fontWeightNormal from './Typography.css';
+  line-height: 2;
   margin-top: -4px;
-  padding: 2px;
-  text-transform: uppercase;
+  text-align: center;
   white-space: nowrap;
 }
 
+.md {
```

Comment:
@denamwangi Can we remove this class? We want to avoid setting specific widths and heights on elements - for i18n reasons. Also let's keep the `font-size` in `.Badge` since we only have 1 size.

![image](https://user-images.githubusercontent.com/127199/81118353-8cd66980-8edd-11ea-8939-f3ee6c826e7e.png)
![image](https://user-images.githubusercontent.com/127199/81118377-94960e00-8edd-11ea-8bc7-724578ae845d.png)

Reply:
That's good to know!